src/maps/mod.rs

```diff
@@ -317,20 +331,18 @@ impl<'a> PerfEventIterator<'a> {
 
         let data_tail = unsafe { (*metadata).data_tail };
 
-        let mmap_size = map.page_count * map.page_size;
+        let mmap_size = map.buffer_size;
```

Comment:
This is the `mmap_size` I referred to that seems potentially confusing since it's 1 page less than the full size of the mmap. Regardless, the code all looks like it's doing the right thing, it's only the name that seems odd.